gui/widgets/qopenglmousetracker.cpp

```diff
@@ -12,7 +12,18 @@ QOpenGLMouseTracker::QOpenGLMouseTracker(float lineWidth,
     : m_camera(cameraFOV, static_cast<float>(size().width()),
                static_cast<float>(size().height()), cameraNearZ, cameraFarZ),
       m_lineWidth(lineWidth), m_lineSelectPrecision(lineSelectPrecision),
-      m_selectedObjectColor(selectedObjectColor), m_frameRate(frameRate) {}
+      m_selectedObjectColor(selectedObjectColor), m_frameRate(frameRate) {
```

Comment:
I think you should be able to just add this to the initialiser list:
`backgroundColor(palette().color(QWidget::backgroundRole()))`
Also shouldn't it be `m_backgroundColor` to match your naming convention here?